examples/ios/README.md

```diff
@@ -0,0 +1,20 @@
+# iOS Sample App
+
+This app demonstrates streaming from the microphone using [AVAudioEngine](https://developer.apple.com/documentation/avfoundation/avaudioengine).
+* Click the globe icon in the upper left corner to enter the URL of cubic server and whether it requires a secure connection (TLS).
+* Click the settings icon in the upper right corner to select from the models available on the server.
+* When the microphone button is held, it streams the audio to the server, and when the button is released, it prints the results to a text area on the screen. 
+
+## Code structure
+Open this entire directory in XCode to build the sample app.  It will load its dependencies automatically, including the Cubic Swift SDK.
```

Comment:
My markdown linter tells me I should always have an extra newline between the `## headers` and the content that follows.

Do we have a style we want to enforce, or is that something we don't really care about?

Reply:
I don't think we've established a style guide for Markdown.  I'm fine with adding a newline if that makes it easier to read the code.

What linter do you use?